pkgs/top-level/qt6-packages.nix

```diff
@@ -27,7 +27,10 @@ makeScopeWithSplicing' {
     inherit (self) callPackage;
     noExtraAttrs = set: lib.attrsets.removeAttrs set [ "extend" "override" "overrideScope" "overrideDerivation" ];
   in (noExtraAttrs qt6) // {
-  inherit stdenv;
+  stdenv =
+    lib.warnIf (lib.oldestSupportedReleaseIsAtLeast 2411)
```

Comment:
24.11 still supports 10.12, so this advice is only wholly applicable for 25.05 and higher; I think you want `2505` here.

OTOH, most Qt‐using applications don’t need the macOS 11 SDK anyway, and those that do should just use `apple-sdk_11` directly, even on 24.11.

Reply:
I think `2411` actually works, because for 24.11 branches `lib.trivial.oldestSupportedRelease` should be `2405`...

Reply:
I guess I don’t understand `oldestSupportedReleaseIsAtLeast`. Isn’t this just `true` unconditionally, even right now? I thought the idea was that you’d write something that becomes `true` after release so that unstable doesn’t throw up warnings that can’t be fixed on supported stable versions (so I assumed `2505` was the right thing).

Reply:
That's true, I'll just put a `lib.warn`.